index.html

```diff
@@ -4102,6 +4102,11 @@ <h3>Co-signed</h3>
               <td>Individual</td>
               <td>DevOps; SRE; Development; open-source community efforts</td>
             </tr>
+      	    <tr>
+              <td><a href="https://github.com/mattcharlton">Matt Charlton/a></td>
```

Comment:
Hey man I think you accidentally deleted the (`<`) of the `</a>` at your name and github link.

```suggestion
              <td><a href="https://github.com/mattcharlton">Matt Charlton</a></td>
```